data/transactions/logic/crypto_test.go

```diff
@@ -117,9 +117,9 @@ func TestVrfVerify(t *testing.T) {
 	testApp(t, notrack("byte 0x1122; byte 0x2233; int 3; vrf_verify VrfAlgorand"), ep, "arg 2 wanted")
 
 	ep = defaultSigParams()
-	testLogic(t, "byte 0x1122; byte 0x2233; byte 0x3344; vrf_verify VrfAlgorand", LogicVersion, ep, "vrf proof wrong size")
+	testLogic(t, notrack("byte 0x1122; byte 0x2233; byte 0x3344; vrf_verify VrfAlgorand"), LogicVersion, ep, "vrf proof wrong size")
```

Comment:
why notrack?

Reply:
Because now that the opcode prototype is more precise, the assembler would normally reject a program that provable gives the wrong sized inputs to `vrf_verify`. `notrack` turns off type tracking, so we can assemble it anyway, and show it fails at eval time.